pyvista/core/filters/data_set.py

```diff
@@ -2903,7 +3423,7 @@ def merge(dataset, grid=None, merge_points=True, inplace=False,
 
         Returns
         -------
-        merged_grid : vtk.UnstructuredGrid
+        vtk.UnstructuredGrid
             Merged grid.
 
         Notes
```

Comment:
Should we also mention the `+` operator here?

Reply:
Will mention.